marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-vis/graph-vis.tsx

```diff
@@ -124,21 +117,49 @@ const GraphVis: React.FC<Props> = (props) => {
     return result;
   };
 
-  const saveUnsavedCoords = () => {
-    // TODO use endpoint that saves entire updated model at once
+  const saveUnsavedCoords = async () => {
     if (props.entityTypes) {
       let newCoords = {...coords};
+      let updatedEntityDefinitions: any = [];
       props.entityTypes.forEach(ent => {
         if (!coordsExist(ent.entityName)) {
           let positions = network.getPositions([ent.entityName])[ent.entityName];
           newCoords[ent.entityName] = {graphX: positions.x, graphY: positions.y};
-          props.saveEntityCoords(ent.entityName, positions.x, positions.y);
+          let entityDefinition: EntityModified = getDefinitionsPayload(ent.entityName, {graphX: positions.x, graphY: positions.y}, ent);
```

Comment:
Suggestion: if you need to have ent and ent.entityName, you can just have ent and use ent.entityName inside the function.

Reply:
Done.